test/spec/storageSrv/storageFirebaseAdapterSrv.spec.js

```diff
@@ -0,0 +1,89 @@
+describe('testing service "storageFirebaseAdapter":', function () {
+    'use strict';
+
+    beforeEach(function(){
+        window.Firebase = function(path){
```

Comment:
why not use MockFireBase....very easy to use
https://github.com/katowulf/mockfirebase